### ISSUE TITLE
A version of Weddase Maryam 

### ISSUE THREAD

**Issue by DenisNosnitsin1970** [BetaMasaheft/Documentation#2285]
I found that there is only one record for Weddase Maryam, LIT2509Weddas, a very common work that is included in the Ethiopic Psalter. Unless I overlooked, there is no other record for an older version of the work, which appears in a few old Psalters, especially the one that stards with the part for Sanbata Krestiyan (as opposed to Senuy) in the common version. The Record in Bm quotes EAE, sayin ghtat " this difference does not constitute two textual traditions", but somehow it is strange, because 1) the difference does produce a separate reduction 2) for sure it appeared not by chance, but was a product of some developments, and it was later substituted for 
the common version also not by chance. I would say, we could have at least an ID for all versions of Weddase Maryam that differ from the common one. SHould I create one?

**Comment by MarcinKrawczuk** [comment]
@DenisNosnitsin1970 could you please give some examples of mss. displaying the "older version", it would be very interesting to see the text.

**Comment by DenisNosnitsin1970** [comment]
R. Cowley remarks in the description of the Psalter British and Foreign Bible Society Mss 176 (p. 92 of the catalogue):
![grafik](https://user-images.githubusercontent.com/18533839/205499355-f6c00eb6-23d9-41bd-8345-ec45f3683b5a.png).
You can check at least BNF Abbadie 105, it's Weddase has a more complex structure then the common version and is quite different fron it; the common beginning of the reading for Monday ፈቀደ ... ያግዕዞ ለአደም is in the midst, fol. 169r. [https://gallica.bnf.fr/ark:/12148/btv1b10093355x/f197.item.r=Abbadie%20105]. EMML 2064 (accessible online) includes Weddase Maryam that starts from Monday, but it is not the common version.

**Comment by MarcinKrawczuk** [comment]
In that case I think it would be advisable to describe in the work record for future catalogers how to distinguish the "old" version from the "new" one.

**Comment by eu-genia** [comment]
We had discussed the Sun/Mon start in the past, we had a meeting early on in the project, and it was decided at that point that sub-`msItem`s where possible are needed to encode the sequence of the days. There are `div`s under the work record to enable this.

**Comment by DenisNosnitsin1970** [comment]
Yes, we can keep, but but the division of the days is only one feature. It is difficult to describe the  internal structure of the old WeMa only by means of reshuffling the days positions. It may be a bit of simplification. Don't we have a similar system for other works: one standard version and one record for all divergent versions?

**Comment by SophiaD-M** [comment]
Dear all,
I would follow Denis and think the "old WeMa" would deserve its own ID because it is much more than just a rearrangement of days.

**Comment by thea-m** [comment]
If the differences go beyond the starting day I would also be in favour of a new record (if it is only the arrangement of days, as Zhenia said, the current record should be enough)

**Comment by thea-m** [comment]
Has this record been created?

**Comment by DenisNosnitsin1970** [comment]
Sorry, as soon as I am done with workshop etc.

**Comment by eu-genia** [comment]
@DenisNosnitsin1970 this is still pending?

**Comment by DenisNosnitsin1970** [comment]
Yes, unfortunately, it is.
